packages/alfa-rules/src/sia-r71/rule.ts

```diff
@@ -13,13 +13,19 @@ import { Scope } from "../tags";
 
 const { hasRole } = DOM;
 const { and } = Predicate;
-const { isVisible } = Style;
+const { isVisible, hasComputedStyle } = Style;
 
 export default Rule.Atomic.of<Page, Element>({
   uri: "https://alfa.siteimprove.com/rules/sia-r71",
   requirements: [Criterion.of("1.4.8")],
   tags: [Scope.Component],
   evaluate({ device, document }) {
+    const isNotJustified = hasComputedStyle(
+      "text-align",
+      (align) => align.value !== "justify",
+      device
+    );
```

Comment:
That can live out of the builder, so it is instancied once when the file is loaded, and not every time the rule is evaluated.

Reply:
It depends on `device` which is only available inside `evaluate`. We could have it as a function on the file level, but that would still initialize a new predicate on each invocation I think 🤔

Reply:
Ah, yes. We do tend to have these out of the `evaluate`, though (as functions `Device => Predicate<Element>`).